### PR TITLE
Release changelog for 2.26

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+
+  [ Kyle Fazzari ]
+  * schema: print allowed length for length failures. (#1056)
+  * tour: add g++ as dependency to 01-reusable-part. (#1062)
+  * file_utils: copy symlinks to directories as symlinks. (#1063)
+  * repo: add multiarch support for stage packages. (#1050)
+  * catkin plugin: don't stage compilers. (#1070)
+  * sources: preserve symlinks to directories. (#1081)
+
+  [ Olivier ]
```
Comment (kyrofa) [inline debian/changelog]: Do we want a last name, here?

Comment (kyrofa) [inline debian/changelog]: Ah, it should probably be merged with his other section below.

Comment (sergiusens) [inline debian/changelog]: done

**Comment by elopio** [comment]
all green \o/

**Comment by sergiusens** [comment]
@elopio zesty integration tests: `sudo: no tty present and no askpass program specified`
https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-zesty/zesty/armhf/s/snapcraft/20170128_105747_cc6d5@/log.gz

**Pull request merged**